tests/prague/eip7685_general_purpose_el_requests/conftest.py

```diff
@@ -50,6 +58,26 @@ def exception() -> BlockException | None:
     return None
 
 
+@pytest.fixture
+def engine_api_error_code(
+    block_body_override_requests: List[Bytes | SupportsBytes] | None,
+) -> EngineAPIError | None:
+    """Engine API error code if any."""
+    if block_body_override_requests is None:
+        return None
+    block_body_override_requests_bytes = [bytes(r) for r in block_body_override_requests]
+    if any(len(r) <= 1 for r in block_body_override_requests_bytes):
+        return EngineAPIError.InvalidParams
+
+    last_type: int = -1
+    for r in block_body_override_requests_bytes:
+        if r[0] <= last_type:
+            return EngineAPIError.InvalidParams
+        last_type = r[0]
```

Comment:
How about this for readability?
```suggestion
    def is_monotonically_increasing(requests: List[bytes]) -> bool:
        return all(x[0] < y[0] for x, y in zip(requests, requests[1:], strict=False))

    if not is_monotonically_increasing(block_body_override_requests_bytes):
        return EngineAPIError.InvalidParams
```

Reply:
This change results in no changes in fixtures to [5424d1b](https://github.com/ethereum/execution-spec-tests/pull/1097/commits/5424d1b93dcc2067210a851cef14a73a18837da1).

Reply:
Awesome, yes this is cleaner 👍